### PR TITLE
changes to chapter 4

### PR DESCRIPTION
**Pull request by jeanetteclark**
make the package updating process more clear

### CODE REVIEW

File: workflows/edit_data_packages/update_an_object.Rmd [hunk 1]
```diff
@@ -1,39 +1,31 @@
...
 You will need to be explicit about your `format_id` here based on the file type. A list of format IDs can be found [here](https://cn.dataone.org/cn/v2/formats) on the DataONE website.
 Use line 2 (Id:) exactly, character for character.
 ```
+
+
+To acomplish the second task, you will need to update the metadata using the EML package. This is covered in [Chapter 6](https://nceas.github.io/datateam-training/training/editing-eml-in-r.html). After you update a file, you will always need to update the metadata because parts of the `physical` section (such as the file size, checksum) will be different, and it may also require different attribute information.
+
+Once you have updated your metadata and saved it, you can update the package itself.
+
+Finally,
```
Comment (dmullen17) [inline workflows/edit_data_packages/update_an_object.Rmd]: trails off here

File: workflows/edit_data_packages/update_an_object.Rmd [hunk 2]
```diff
@@ -1,39 +1,31 @@
...
-
-file.create("new_file.csv")
-update_package_object(mnT, pkg$data[1], "new_file.csv", pkg$resource_map, format_id = "text/csv")
-file.remove("new_file.csv") #clean up
+The code will look like this (very similar to the `publish_object` function call):
 
-#check out the package on test.arcticdata.io
+```{r, eval = F}
+id_new <- update_object(mnT, 
+                        data_pid = "the data pid you need to update", #pkg$data
```
Comment (dmullen17) [inline workflows/edit_data_packages/update_an_object.Rmd]: I'd remove `#pkg$data` since that can be one or more pids

**Comment by dmullen17** [comment]
We don't have to add this in to the training now.  But it would probably help to show an example eml physical update, and then updating the package with the new eml under the `metadata_path` argument, and adding/removing the new / old data pids from the `data_pids` argument, along with the rest of the non-updated data pids

File: workflows/edit_data_packages/update_a_package.Rmd [hunk 3]
```diff
@@ -1,50 +1,36 @@
...
-write_eml(eml, eml_path)
-```
+Now we can update your data package.
 
-Next verify the eml is valid.
+```{r, eval = F}
+eml_path <- "path/to/your/saved/eml.xml"
 
-``` {r, eval = F}
-EML::eml_validate(eml_path)
```
Comment (emilyodean) [inline workflows/edit_data_packages/update_a_package.Rmd]: Is this discussed later in the training?

Comment (jeanetteclark) [reply workflows/edit_data_packages/update_a_package.Rmd]: it was discussed previously, but those lines were removed not added

Comment (emilyodean) [reply workflows/edit_data_packages/update_a_package.Rmd]: Yep, saw that they were removed and just wanted to make sure we had it in there somewhere.

**Comment by jeanetteclark** [comment]
@dmullen17 I agree. let's plan on doing that after the reshuffle

**Pull request merged**